### ISSUE TITLE
Right click spamming causes tp'ing and other issues

### ISSUE THREAD

**Issue by nomadjimbob** [Drustcraft/Drustcraft-Game#11]
This is being caused by AAC and has been confirmed by myself. Disabling the AAC plugin stops this from occurring.

https://github.com/konsolas/AAC-Issues/issues/2295

**Comment by lstone01** [comment]
Is this the thing that happened with the compass?

**Comment by nomadjimbob** [comment]
No. You can see it in action in the linked issue with AAC - https://streamable.com/xbcjqd
